src/style.css

```diff
@@ -1,3 +1,58 @@
 body {
+  text-align: center;
   margin: 0;
+  font-family: "Roboto", sans-serif;
+  background-color: lightslategray;
+}
+
+.header {
+  width: 100%;
+  padding: 0 50px;
+  background-color: whitesmoke;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.nav {
+  height: 60px;
+  align-items: center;
+}
+
+.nav-list {
+  list-style: none;
+  display: flex;
+  align-items: center;
+  object-fit: contain;
+  padding: 0;
+}
+
+.nav-item {
+  margin: 0 20px;
+}
+
+.nav-link {
+  text-decoration: none;
+  color: #000;
+  font-weight: bold;
+}
+
+.nav-link:hover {
+  color: var(--is-active-clor, #00acdc);
+}
+
+.is-active {
+  position: relative;
+}
+
+.is-active::after {
+  height: 4px;
+  width: 100%;
+  content: "";
+  background-color: #00acdc;
+  position: absolute;
+  bottom: -130%;
```

Comment:
```suggestion
  bottom: 0;
```

Reply:
How to place the blue line at the bottom of header with not using bottom -130%?